frame/balances/src/lib.rs

```diff
@@ -15,17 +15,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! # Balances Module
+//! # Balances Pallet
 //!
-//! The Balances module provides functionality for handling accounts and balances.
+//! The Balances pallet provides functionality for handling accounts and balances.
 //!
 //! - [`balances::Config`](./trait.Config.html)
 //! - [`Call`](./enum.Call.html)
-//! - [`Module`](./struct.Module.html)
+//! - [`Pallet`](./struct.Pallet.html)
```

Comment:
This link is not right. Maybe all of these can just be `[Thing]` now?

Reply:
yeah all these "old way" links could be updated.